tests/schema/tests.py

```diff
@@ -5298,3 +5303,29 @@ def test_ci_cs_db_collation(self):
             if connection.vendor == "postgresql":
                 with connection.cursor() as cursor:
                     cursor.execute("DROP COLLATION IF EXISTS case_insensitive")
+
+
+@isolate_apps("schema")
+@skipUnlessAnyDBFeature("supports_json_field", "supports_stored_generated_columns")
+class GeneratedFieldTransactionTests(TransactionTestCase):
+    available_apps = ["schema"]
+
+    def test_generated_sql(self):
+        """
+        Regression test for #34861
+
+        GeneratedField.generated_sql shouldn't generate broken SQL code.
+        """
+        from django.db.models import GeneratedField, JSONField
+        from django.db.models.fields.json import KT
```

Comment:
Inline imports are not necessary.

Reply:
Your message is a little too vague or polite. :)

Are you just commenting that inline imports aren't necessary or are you asking me to move these imports up for you to consider the PR mergeable?

Reply:
:smile: Yes, please move them to the top :100:

Reply:
Done :)